[2023-03-07 08:40:34,782] {processor.py:153} INFO - Started process (PID=7949) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:40:34,782] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:40:34,783] {logging_mixin.py:115} INFO - [2023-03-07 08:40:34,782] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:40:34,796] {logging_mixin.py:115} INFO - [2023-03-07 08:40:34,796] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:40:34,796] {logging_mixin.py:115} INFO - [2023-03-07 08:40:34,796] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:40:34,796] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:40:34,803] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-03-07 08:41:10,528] {processor.py:153} INFO - Started process (PID=8207) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:10,529] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:41:10,531] {logging_mixin.py:115} INFO - [2023-03-07 08:41:10,531] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:10,560] {logging_mixin.py:115} INFO - [2023-03-07 08:41:10,559] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:41:10,560] {logging_mixin.py:115} INFO - [2023-03-07 08:41:10,560] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:41:10,560] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:10,565] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-03-07 08:41:41,136] {processor.py:153} INFO - Started process (PID=8274) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:41,137] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:41:41,138] {logging_mixin.py:115} INFO - [2023-03-07 08:41:41,138] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:41,166] {logging_mixin.py:115} INFO - [2023-03-07 08:41:41,165] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:41:41,166] {logging_mixin.py:115} INFO - [2023-03-07 08:41:41,166] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:41:41,166] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:41:41,172] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:42:13,595] {processor.py:153} INFO - Started process (PID=8341) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:13,596] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:42:13,596] {logging_mixin.py:115} INFO - [2023-03-07 08:42:13,596] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:13,611] {logging_mixin.py:115} INFO - [2023-03-07 08:42:13,611] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:42:13,611] {logging_mixin.py:115} INFO - [2023-03-07 08:42:13,611] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:42:13,611] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:13,617] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-03-07 08:42:45,600] {processor.py:153} INFO - Started process (PID=8405) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:45,600] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:42:45,601] {logging_mixin.py:115} INFO - [2023-03-07 08:42:45,601] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:45,621] {logging_mixin.py:115} INFO - [2023-03-07 08:42:45,620] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:42:45,621] {logging_mixin.py:115} INFO - [2023-03-07 08:42:45,621] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:42:45,621] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:42:45,629] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-03-07 08:43:16,584] {processor.py:153} INFO - Started process (PID=8478) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:16,586] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:43:16,587] {logging_mixin.py:115} INFO - [2023-03-07 08:43:16,587] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:16,617] {logging_mixin.py:115} INFO - [2023-03-07 08:43:16,617] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:43:16,617] {logging_mixin.py:115} INFO - [2023-03-07 08:43:16,617] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:43:16,618] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:16,624] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-03-07 08:43:48,765] {processor.py:153} INFO - Started process (PID=8541) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:48,766] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:43:48,767] {logging_mixin.py:115} INFO - [2023-03-07 08:43:48,767] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:48,794] {logging_mixin.py:115} INFO - [2023-03-07 08:43:48,794] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:43:48,795] {logging_mixin.py:115} INFO - [2023-03-07 08:43:48,794] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:43:48,795] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:43:48,802] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-03-07 08:44:20,283] {processor.py:153} INFO - Started process (PID=8617) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:20,285] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:44:20,286] {logging_mixin.py:115} INFO - [2023-03-07 08:44:20,286] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:20,314] {logging_mixin.py:115} INFO - [2023-03-07 08:44:20,314] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:44:20,314] {logging_mixin.py:115} INFO - [2023-03-07 08:44:20,314] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:44:20,314] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:20,321] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-03-07 08:44:52,047] {processor.py:153} INFO - Started process (PID=8683) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:52,048] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:44:52,049] {logging_mixin.py:115} INFO - [2023-03-07 08:44:52,049] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:52,077] {logging_mixin.py:115} INFO - [2023-03-07 08:44:52,077] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:44:52,077] {logging_mixin.py:115} INFO - [2023-03-07 08:44:52,077] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:44:52,077] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:44:52,086] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-03-07 08:45:22,580] {processor.py:153} INFO - Started process (PID=8741) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:22,580] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:45:22,581] {logging_mixin.py:115} INFO - [2023-03-07 08:45:22,581] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:22,596] {logging_mixin.py:115} INFO - [2023-03-07 08:45:22,596] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:45:22,596] {logging_mixin.py:115} INFO - [2023-03-07 08:45:22,596] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:45:22,596] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:22,601] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-03-07 08:45:53,432] {processor.py:153} INFO - Started process (PID=8813) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:53,433] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:45:53,434] {logging_mixin.py:115} INFO - [2023-03-07 08:45:53,434] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:53,460] {logging_mixin.py:115} INFO - [2023-03-07 08:45:53,460] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:45:53,461] {logging_mixin.py:115} INFO - [2023-03-07 08:45:53,461] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:45:53,461] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:45:53,468] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:46:25,284] {processor.py:153} INFO - Started process (PID=8892) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:25,285] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:46:25,286] {logging_mixin.py:115} INFO - [2023-03-07 08:46:25,285] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:25,302] {logging_mixin.py:115} INFO - [2023-03-07 08:46:25,302] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:46:25,302] {logging_mixin.py:115} INFO - [2023-03-07 08:46:25,302] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:46:25,302] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:25,310] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-03-07 08:46:56,082] {processor.py:153} INFO - Started process (PID=8971) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:56,083] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:46:56,084] {logging_mixin.py:115} INFO - [2023-03-07 08:46:56,084] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:56,110] {logging_mixin.py:115} INFO - [2023-03-07 08:46:56,109] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:46:56,110] {logging_mixin.py:115} INFO - [2023-03-07 08:46:56,110] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:46:56,110] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:46:56,117] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:47:28,909] {processor.py:153} INFO - Started process (PID=9048) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:47:28,910] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:47:28,911] {logging_mixin.py:115} INFO - [2023-03-07 08:47:28,911] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:47:28,937] {logging_mixin.py:115} INFO - [2023-03-07 08:47:28,937] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:47:28,938] {logging_mixin.py:115} INFO - [2023-03-07 08:47:28,937] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:47:28,938] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:47:28,946] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:48:01,479] {processor.py:153} INFO - Started process (PID=9146) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:01,481] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:48:01,482] {logging_mixin.py:115} INFO - [2023-03-07 08:48:01,481] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:01,501] {logging_mixin.py:115} INFO - [2023-03-07 08:48:01,501] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:48:01,501] {logging_mixin.py:115} INFO - [2023-03-07 08:48:01,501] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:48:01,501] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:01,509] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-03-07 08:48:32,265] {processor.py:153} INFO - Started process (PID=9227) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:32,266] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:48:32,267] {logging_mixin.py:115} INFO - [2023-03-07 08:48:32,267] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:32,281] {logging_mixin.py:115} INFO - [2023-03-07 08:48:32,281] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:48:32,281] {logging_mixin.py:115} INFO - [2023-03-07 08:48:32,281] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:48:32,282] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:48:32,289] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-03-07 08:49:03,128] {processor.py:153} INFO - Started process (PID=9295) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:03,130] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:49:03,131] {logging_mixin.py:115} INFO - [2023-03-07 08:49:03,130] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:03,151] {logging_mixin.py:115} INFO - [2023-03-07 08:49:03,151] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:49:03,151] {logging_mixin.py:115} INFO - [2023-03-07 08:49:03,151] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:49:03,152] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:03,159] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-03-07 08:49:36,205] {processor.py:153} INFO - Started process (PID=9389) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:36,206] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:49:36,207] {logging_mixin.py:115} INFO - [2023-03-07 08:49:36,207] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:36,226] {logging_mixin.py:115} INFO - [2023-03-07 08:49:36,226] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:49:36,226] {logging_mixin.py:115} INFO - [2023-03-07 08:49:36,226] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:49:36,226] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:49:36,235] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-03-07 08:50:08,125] {processor.py:153} INFO - Started process (PID=9492) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:08,126] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:50:08,127] {logging_mixin.py:115} INFO - [2023-03-07 08:50:08,127] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:08,154] {logging_mixin.py:115} INFO - [2023-03-07 08:50:08,153] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:50:08,154] {logging_mixin.py:115} INFO - [2023-03-07 08:50:08,154] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:50:08,154] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:08,164] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:50:38,989] {processor.py:153} INFO - Started process (PID=9567) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:38,990] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:50:38,991] {logging_mixin.py:115} INFO - [2023-03-07 08:50:38,991] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:39,008] {logging_mixin.py:115} INFO - [2023-03-07 08:50:39,008] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:50:39,009] {logging_mixin.py:115} INFO - [2023-03-07 08:50:39,009] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:50:39,009] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:50:39,040] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2023-03-07 08:51:11,305] {processor.py:153} INFO - Started process (PID=9645) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:11,306] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:51:11,308] {logging_mixin.py:115} INFO - [2023-03-07 08:51:11,307] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:11,333] {logging_mixin.py:115} INFO - [2023-03-07 08:51:11,333] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:51:11,333] {logging_mixin.py:115} INFO - [2023-03-07 08:51:11,333] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:51:11,333] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:11,341] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 08:51:42,368] {processor.py:153} INFO - Started process (PID=9753) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:42,403] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:51:42,403] {logging_mixin.py:115} INFO - [2023-03-07 08:51:42,403] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:42,417] {logging_mixin.py:115} INFO - [2023-03-07 08:51:42,417] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:51:42,417] {logging_mixin.py:115} INFO - [2023-03-07 08:51:42,417] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:51:42,418] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:51:42,428] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2023-03-07 08:52:16,211] {processor.py:153} INFO - Started process (PID=9839) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:16,212] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:52:16,213] {logging_mixin.py:115} INFO - [2023-03-07 08:52:16,213] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:16,229] {logging_mixin.py:115} INFO - [2023-03-07 08:52:16,228] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:52:16,229] {logging_mixin.py:115} INFO - [2023-03-07 08:52:16,229] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:52:16,229] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:16,236] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-03-07 08:52:50,512] {processor.py:153} INFO - Started process (PID=9931) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:50,513] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:52:50,513] {logging_mixin.py:115} INFO - [2023-03-07 08:52:50,513] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:50,523] {logging_mixin.py:115} INFO - [2023-03-07 08:52:50,523] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:52:50,523] {logging_mixin.py:115} INFO - [2023-03-07 08:52:50,523] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:52:50,523] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:52:50,529] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-03-07 08:53:23,983] {processor.py:153} INFO - Started process (PID=10017) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:23,984] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:53:23,985] {logging_mixin.py:115} INFO - [2023-03-07 08:53:23,985] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:24,011] {logging_mixin.py:115} INFO - [2023-03-07 08:53:24,010] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:53:24,011] {logging_mixin.py:115} INFO - [2023-03-07 08:53:24,011] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:53:24,011] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:24,020] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-03-07 08:53:54,732] {processor.py:153} INFO - Started process (PID=10088) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:54,733] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:53:54,733] {logging_mixin.py:115} INFO - [2023-03-07 08:53:54,733] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:54,743] {logging_mixin.py:115} INFO - [2023-03-07 08:53:54,743] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:53:54,743] {logging_mixin.py:115} INFO - [2023-03-07 08:53:54,743] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:53:54,744] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:53:54,748] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-03-07 08:54:23,002] {processor.py:153} INFO - Started process (PID=10245) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:23,003] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:54:23,003] {logging_mixin.py:115} INFO - [2023-03-07 08:54:23,003] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:23,015] {logging_mixin.py:115} INFO - [2023-03-07 08:54:23,015] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:54:23,015] {logging_mixin.py:115} INFO - [2023-03-07 08:54:23,015] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:54:23,016] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:23,021] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-03-07 08:54:53,564] {processor.py:153} INFO - Started process (PID=10854) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:53,565] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:54:53,567] {logging_mixin.py:115} INFO - [2023-03-07 08:54:53,567] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:53,599] {logging_mixin.py:115} INFO - [2023-03-07 08:54:53,599] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:54:53,600] {logging_mixin.py:115} INFO - [2023-03-07 08:54:53,600] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:54:53,600] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:54:53,610] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2023-03-07 08:55:27,705] {processor.py:153} INFO - Started process (PID=10933) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:27,707] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:55:27,709] {logging_mixin.py:115} INFO - [2023-03-07 08:55:27,708] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:27,729] {logging_mixin.py:115} INFO - [2023-03-07 08:55:27,728] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:55:27,729] {logging_mixin.py:115} INFO - [2023-03-07 08:55:27,729] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:55:27,729] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:27,736] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-03-07 08:55:59,596] {processor.py:153} INFO - Started process (PID=10998) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:59,598] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 08:55:59,599] {logging_mixin.py:115} INFO - [2023-03-07 08:55:59,599] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:59,620] {logging_mixin.py:115} INFO - [2023-03-07 08:55:59,620] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 08:55:59,620] {logging_mixin.py:115} INFO - [2023-03-07 08:55:59,620] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 08:55:59,620] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 08:55:59,628] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-03-07 09:01:42,496] {processor.py:153} INFO - Started process (PID=11525) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:01:42,497] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:01:42,497] {logging_mixin.py:115} INFO - [2023-03-07 09:01:42,497] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:01:42,508] {logging_mixin.py:115} INFO - [2023-03-07 09:01:42,508] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:01:42,508] {logging_mixin.py:115} INFO - [2023-03-07 09:01:42,508] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:01:42,508] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:01:42,513] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-03-07 09:02:14,344] {processor.py:153} INFO - Started process (PID=11918) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:14,344] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:02:14,344] {logging_mixin.py:115} INFO - [2023-03-07 09:02:14,344] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:14,357] {logging_mixin.py:115} INFO - [2023-03-07 09:02:14,356] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:02:14,357] {logging_mixin.py:115} INFO - [2023-03-07 09:02:14,357] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:02:14,357] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:14,364] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-03-07 09:02:45,163] {processor.py:153} INFO - Started process (PID=11998) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:45,164] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:02:45,165] {logging_mixin.py:115} INFO - [2023-03-07 09:02:45,164] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:45,181] {logging_mixin.py:115} INFO - [2023-03-07 09:02:45,181] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:02:45,181] {logging_mixin.py:115} INFO - [2023-03-07 09:02:45,181] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:02:45,182] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:02:45,189] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-03-07 09:03:15,958] {processor.py:153} INFO - Started process (PID=12081) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:15,959] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:03:15,960] {logging_mixin.py:115} INFO - [2023-03-07 09:03:15,960] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:15,984] {logging_mixin.py:115} INFO - [2023-03-07 09:03:15,984] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:03:15,984] {logging_mixin.py:115} INFO - [2023-03-07 09:03:15,984] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:03:15,984] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:15,993] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-03-07 09:03:46,790] {processor.py:153} INFO - Started process (PID=12236) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:46,790] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:03:46,791] {logging_mixin.py:115} INFO - [2023-03-07 09:03:46,791] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:46,804] {logging_mixin.py:115} INFO - [2023-03-07 09:03:46,803] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:03:46,804] {logging_mixin.py:115} INFO - [2023-03-07 09:03:46,804] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:03:46,804] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:03:46,810] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-03-07 09:04:20,921] {processor.py:153} INFO - Started process (PID=12315) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:20,922] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:04:20,923] {logging_mixin.py:115} INFO - [2023-03-07 09:04:20,923] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:20,948] {logging_mixin.py:115} INFO - [2023-03-07 09:04:20,948] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:04:20,948] {logging_mixin.py:115} INFO - [2023-03-07 09:04:20,948] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:04:20,948] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:20,956] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-03-07 09:04:52,247] {processor.py:153} INFO - Started process (PID=12395) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:52,248] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:04:52,248] {logging_mixin.py:115} INFO - [2023-03-07 09:04:52,248] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:52,257] {logging_mixin.py:115} INFO - [2023-03-07 09:04:52,257] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:04:52,257] {logging_mixin.py:115} INFO - [2023-03-07 09:04:52,257] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:04:52,257] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:04:52,263] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-03-07 09:05:22,391] {processor.py:153} INFO - Started process (PID=12465) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:22,392] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:05:22,392] {logging_mixin.py:115} INFO - [2023-03-07 09:05:22,392] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:22,413] {logging_mixin.py:115} INFO - [2023-03-07 09:05:22,413] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:05:22,413] {logging_mixin.py:115} INFO - [2023-03-07 09:05:22,413] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:05:22,414] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:22,423] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-03-07 09:05:54,994] {processor.py:153} INFO - Started process (PID=12537) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:54,996] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:05:54,997] {logging_mixin.py:115} INFO - [2023-03-07 09:05:54,997] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:55,030] {logging_mixin.py:115} INFO - [2023-03-07 09:05:55,030] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:05:55,031] {logging_mixin.py:115} INFO - [2023-03-07 09:05:55,031] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:05:55,031] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:05:55,041] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2023-03-07 09:06:27,324] {processor.py:153} INFO - Started process (PID=12603) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:27,325] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:06:27,327] {logging_mixin.py:115} INFO - [2023-03-07 09:06:27,326] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:27,357] {logging_mixin.py:115} INFO - [2023-03-07 09:06:27,357] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:06:27,357] {logging_mixin.py:115} INFO - [2023-03-07 09:06:27,357] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:06:27,357] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:27,366] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2023-03-07 09:06:58,725] {processor.py:153} INFO - Started process (PID=12670) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:58,726] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:06:58,727] {logging_mixin.py:115} INFO - [2023-03-07 09:06:58,727] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:58,754] {logging_mixin.py:115} INFO - [2023-03-07 09:06:58,753] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:06:58,754] {logging_mixin.py:115} INFO - [2023-03-07 09:06:58,754] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:06:58,754] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:06:58,762] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-03-07 09:07:30,276] {processor.py:153} INFO - Started process (PID=12745) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:07:30,278] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:07:30,279] {logging_mixin.py:115} INFO - [2023-03-07 09:07:30,279] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:07:30,314] {logging_mixin.py:115} INFO - [2023-03-07 09:07:30,314] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:07:30,315] {logging_mixin.py:115} INFO - [2023-03-07 09:07:30,315] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:07:30,315] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:07:30,324] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2023-03-07 09:08:00,799] {processor.py:153} INFO - Started process (PID=12860) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:00,801] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:08:00,802] {logging_mixin.py:115} INFO - [2023-03-07 09:08:00,802] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:00,818] {logging_mixin.py:115} INFO - [2023-03-07 09:08:00,818] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:08:00,818] {logging_mixin.py:115} INFO - [2023-03-07 09:08:00,818] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:08:00,818] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:00,824] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-03-07 09:08:31,094] {processor.py:153} INFO - Started process (PID=12926) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:31,095] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:08:31,096] {logging_mixin.py:115} INFO - [2023-03-07 09:08:31,095] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:31,123] {logging_mixin.py:115} INFO - [2023-03-07 09:08:31,123] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:08:31,123] {logging_mixin.py:115} INFO - [2023-03-07 09:08:31,123] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:08:31,123] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:08:31,132] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-03-07 09:09:01,366] {processor.py:153} INFO - Started process (PID=13005) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:01,367] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:09:01,367] {logging_mixin.py:115} INFO - [2023-03-07 09:09:01,367] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:01,383] {logging_mixin.py:115} INFO - [2023-03-07 09:09:01,383] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:09:01,384] {logging_mixin.py:115} INFO - [2023-03-07 09:09:01,384] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:09:01,384] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:01,392] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-03-07 09:09:33,877] {processor.py:153} INFO - Started process (PID=13088) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:33,879] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:09:33,880] {logging_mixin.py:115} INFO - [2023-03-07 09:09:33,880] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:33,916] {logging_mixin.py:115} INFO - [2023-03-07 09:09:33,915] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:09:33,916] {logging_mixin.py:115} INFO - [2023-03-07 09:09:33,916] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:09:33,916] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:33,924] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2023-03-07 09:09:45,746] {processor.py:153} INFO - Started process (PID=13147) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:45,747] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 09:09:45,747] {logging_mixin.py:115} INFO - [2023-03-07 09:09:45,747] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:45,757] {logging_mixin.py:115} INFO - [2023-03-07 09:09:45,757] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 09:09:45,758] {logging_mixin.py:115} INFO - [2023-03-07 09:09:45,758] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 09:09:45,758] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 09:09:45,762] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
